### PR TITLE
Qianyu

### PR DESCRIPTION
**Pull request by qianyyu**

### CODE REVIEW

File: users/templates/users/auth_test.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block title %}PolarBear -- sign up{% endblock %}
+
+{% block content %}
+<p>Login/Signup Successfully!! You will be redirected to home page in <span id="sp">4</span> seconds...</p>
+<script>
+
```
Comment (petelliott) [inline users/templates/users/auth_test.html]: this page wasn't meant to be permanent. when we have a homepage, we should replace it. this file should be deleted.

File: users/views.py [hunk 2]
```diff
@@ -50,7 +58,9 @@ def login(request):
 def authenticated_test(request):
     if request.user.is_authenticated:
         num = request.user.author.number
-        return HttpResponse("your number is {}".format(num))
+        # return HttpResponse("your number is {}".format(num))
+        # return redirect("homepage")
```
Comment (petelliott) [inline users/views.py:130]: this should be reverted, or redirect directly to the homepage

**Review by petelliott — CHANGES_REQUESTED** [review]

File: users/templates/users/auth_test.html (hunk 1 above)
Comment (qianyyu) [reply users/templates/users/auth_test.html]: fixed

File: users/templates/users/create.html [hunk 3]
```diff
@@ -1,11 +1,87 @@
+<style>
```
Comment (petelliott) [inline users/templates/users/create.html]: ideally we should share the css for this and the login page in a static css file.
maybe add a head block to barbase.html?

**Review by petelliott — CHANGES_REQUESTED** [review]

File: users/templates/users/create.html (hunk 3 above)
Comment (qianyyu) [reply users/templates/users/create.html]: Done

**Review by petelliott — APPROVED** [review]

**Pull request merged**